### DISCUSSION TITLE
Unable to bypass branch protection when not repo admin

### DISCUSSION THREAD

**Discussion by anuraaga** [community/community#137586] [Mobile]
### Select Topic Area

Bug

### Body

I am working on a repo that has a branch ruleset on the main branch that restricts updates, and has a bypass list of members, including non-admins. We merge PRs by bypassing protections (this is the only way we found to restrict who can merge PRs in the repo).

I found I am unable to merge at all in the GitHub app, it says "Unable to merge". I know I have permission to bypass since I can open the PR in the mobile browser and bypass+merge. I have seen bypass on mobile app before and believe that is when I am a repo admin, while in this case I am in the bypass list but not a repo admin. I guess not presenting the bypass ui for this case is not intended. At the least, the "Unable to merge" text could be more informative.

**Comment by elisealix22** [comment]
Hi @anuraaga, thank you so much for reporting this! I opened an issue to track this internally and we will look into it. Let us know if you run into any other trouble in the meantime.
